interface-definitions/service_dns_forwarding.xml.in

```diff
@@ -670,6 +670,19 @@
                 </properties>
                 <defaultValue>3600</defaultValue>
               </leafNode>
+              <leafNode name="serve-stale-extensions">
+                <properties>
+                  <help>Maximum number of times an expired record’s TTL is extended by 30s when serving stale</help>
```

Comment:
Please avoid `’` in help string.

`Number of times an expired TTL record is extended by 30 seconds when serving stale`

Reply:
I went with 
```
Number of times the expired TTL of a record is extended by 30 seconds when serving stale
```